documentation/asciidoc/stories/assembly_batch_cr.adoc

```diff
@@ -0,0 +1,15 @@
+ifdef::context[:parent-context: {context}]
+[id='batch_operations']
+:context: batch
+= Running Batch Operations
+{ispn_operator} provides a `Batch` CR that lets you create {brandname} resources in bulk.
```

Comment:
@oraNod I'm not sure, whether we need to identify the list of operations and resources we can utilize Batch CR or not?

Reply:
@dmvolod Interesting thought. I feel like this topic covers all the Infinispan resources: users, caches, counters, schema, tasks/scripts. Those are the things you see in the console. I don't think we should go too far into enumerating operations in operator docs. Operator docs should rely on CLI docs for command details. There's a separate docs issue in this sprint to work on CLI batch docs as well.

Reply:
Oh, yes, we can provide a link to the core docs when will be available.